manifest.json

```diff
@@ -2,7 +2,7 @@
   "name": "@deskpro-apps/google-calendar",
   "title": "Google Calendar",
   "description": "View, create, and schedule future events with your users.",
-  "version": "1.0.1",
```

Comment:
Because there was no release yet and no version has been published yet

Reply:
You do have releases, this will cause contradictions. Each merge to master should show a change in the state.
![IMG_20230511_143034.jpg](https://github.com/DeskproApps/google-calendar/assets/54243490/902cf9c8-45c0-4593-bec2-b560f2f5dcde)